src/renderer/pages/settings-page.component.tsx

```diff
@@ -161,6 +167,8 @@ export function SettingsPage() {
 
   const openLogs = () => ipcService.sendLazy("open-logs");
 
+  const openChangelog = () => setModalChangelogResponse(updaterService.openChangelog())
```

Comment:
you store the result of openChangelog but the function always return a truthy value due to the promise